### PR TITLE
bump to conmon 2.0.11

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

> /kind cleanup

#### What this PR does / why we need it:
bumps conmon dependency

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?
None

```release-note
None
```

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -714,7 +667,6 @@ golang.org/x/crypto/ssh/agent
 golang.org/x/crypto/ssh/knownhosts
 golang.org/x/crypto/ssh/terminal
 # golang.org/x/net v0.0.0-20200226121028-0de0cce0169b
-## explicit
```
Comment (saschagrunert) [inline vendor/modules.txt]: Ugh, I think most of those changes are coming from go versions < 1.14. Can you please revert them?

**Comment by saschagrunert** [comment]
@haircommander we have to remove the `> ` from the kind to be applied correctly:
/kind cleanup

**Comment by rhatdan** [comment]
/test e2e_fedora
LGTM
@saschagrunert @mrunalp @umohnani8 @kolyshkin PTAL

**Comment by rhatdan** [comment]
@haircommander Needs a rebase.

**Pull request merged**